packages/manager/modules/pci/src/projects/project/public-gateways/add/add.controller.js

```diff
@@ -1,19 +1,24 @@
 import get from 'lodash/get';
+import set from 'lodash/set';
```

Comment:
In this case, we store the variable we need to update as a string constant. So let us consider we need to update 'ip.region' on "this" object. If we use this[] notation, a new variable called 'ip.region' is created on this instead of assigning the value to the existing ip property on this. So lodash set method is used to achieve this case.